Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env groovy
+
+@Library('lco-shared-libs@0.0.10') _
+
+pipeline {
+	agent any
+	environment {
+		dockerImage = null
+		PROJ_NAME = projName()
+		GIT_DESCRIPTION = gitDescribe()
+		DOCKER_IMG = dockerImageName("${LCO_DOCK_REG}", "${PROJ_NAME}", "${GIT_DESCRIPTION}")
```

Comment:
Looks like the jenkins build failed due to the github repo name, but then I found this repo? https://github.com/LCOGT/floyds-pipeline
```
[YDS_pipeline_feature_docker-IDYMTBJFGU3WU4ZGJ63SQ2C3SO4KTER5SG7HQSSNTO5UII6HGCIA] Running shell script
+ docker build -t docker.lco.global/FLOYDS_pipeline:v2.2.2-55-gaa11fe5 .
invalid argument "docker.lco.global/FLOYDS_pipeline:v2.2.2-55-gaa11fe5" for "-t, --tag" flag: invalid reference format: repository name must be lowercase
See 'docker build --help'.
script returned exit code 125
```

Reply:
That's gross. I think I am going to delete the other repo. All of that code is now in here.